### ISSUE TITLE
How to check for broken local file links?

### ISSUE THREAD

**Issue by uvsmtid** [tcort/markdown-link-check#463]
I have this configuration:

**`.pre-commit-config.yaml`**

```yaml

repos:
    -   repo: https://github.com/tcort/markdown-link-check
        rev: v3.13.7
        hooks:

        -   id: markdown-link-check
            args:
            -   "-q"
            -   "-c"
            -   ".markdownlinkcheck.json"
```

**`.markdownlinkcheck.json`**

```json
{
  "checkRelative": true
}
```

But local broken links are still not detected:

```markdown
[some_file.md]: this/path/does/not/exists/some_file.md
```

**Comment by hofbi** [comment]
This seems to be a duplicate of https://github.com/tcort/markdown-link-check/issues/445

**Comment by uvsmtid** [comment]
@hofbi, thanks for the pointer!

Indeed, I tested it without `pre-commit` directly and the broken link detection works by default:

```sh
npm install markdown-link-check --save-dev
npx markdown-link-check readme.md
# broken link detected
```

Verified on:

```sh
npx markdown-link-check --version
3.13.7
```

**Comment by uvsmtid** [comment]
Actually, I think the problem is with `markdown-link-check`, not with `pre-commit`.

I realized the exit code of the failed command is still zero:

```sh
# This command reports broken links, but also prints "SUCCESS":

npx markdown-link-check readme.md && echo "SUCCESS"
```

**Comment by hofbi** [comment]
Then https://github.com/tcort/markdown-link-check/issues/457 seems to be the root cause
